mautic/mautic
# TPROD-332 - CitrixBundle - Migrate PHP Templates to Twig Templates

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. 



**REQUIRES** an active Citrix account to test. You will need to connect to it. Enabling the plugins and trying to test will cause failures.

Once Citrix plugins are enabled:
- Test new form fields
- Campaign actions
